src/implementation.cpp

```diff
@@ -28,7 +28,7 @@ template ada::result<url_aggregator> parse<url_aggregator>(
 std::string href_from_file(std::string_view input) {
   // This is going to be much faster than constructing a URL.
   std::string tmp_buffer;
-  std::string_view internal_input;
+  std::string_view internal_input{};
```

Comment:
I don't understand the change here, can you elaborate on your purpose?

Reply:
@mertcanaltin Can you elaborate on why `internal_input` is uninitialized after... this line...

```cpp
std::string href_from_file(std::string_view input) {
  std::string tmp_buffer;
  std::string_view internal_input;
  ...
}
```

This like like textbook C++:

```
class A {...};

void myfunc() {
  A a; // call default constructor of A
}
```

Please don't argue from the warning message of some tool. There is an endless stream of warnings one can get from static analyzers.


The `std::string_view` class has a default constructor.